app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -3,8 +3,51 @@ kind: registration
 name: Global Registration
 model: ProvisioningTemplate
 -%>
+<%
+    curl_opts = ["-H 'Content-Type: application/json'", "-H 'Accept: application/json'", "-H 'Authorization: Bearer #{@auth_token}'"]
+%>
+#!/bin/bash
```

Comment:
If we want this to be truly universal and os-agnostic, shouldn't it use posix sh instead of bash?

Reply:
So `#!/bin/sh` instead of `#!/bin/bash`?